aero-sample-apps/aero-sample-apps

```diff
@@ -0,0 +1,11 @@
+Section: misc
```

Comment:
This indicates that Debian package is stored at the 'misc' folder in the site where we are hosting the package. We need to re-check on this.